lib/EditableList/ItemView.js

```diff
@@ -2,14 +2,15 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import css from './EditableList.css';
 
-const ItemView = ({ cells }) => (
-  <div className={css.editListRow} role="row">
+const ItemView = ({ cells, rowIndex }) => (
+  <div className={css.editListRow} role="row" aria-rowindex={rowIndex + 2}>
     {cells}
   </div>
 );
 
 ItemView.propTypes = {
   cells: PropTypes.arrayOf(PropTypes.object).isRequired,
+  rowIndex: PropTypes.number
```

Comment:
Should this propType be required?

Reply:
👍 I'll make it so. For the most, this comes from MCL, but if it isn't there, the addition would fail...

Reply:
Made it so. Makes sense since the "addition" leans on it.